integration-tests/src/test/java/oracle/kubernetes/operator/BaseTest.java

```diff
@@ -520,4 +583,29 @@ private void callWebAppAndVerifyScaling(Domain domain, int replicas) throws Exce
       TestUtils.checkServiceCreated(podName, domainNS);
     }
   }
+
+  public static void tearDown() throws Exception {
+    StringBuffer cmd =
+        new StringBuffer("export RESULT_ROOT=$RESULT_ROOT && export PV_ROOT=$PV_ROOT && ");
+    cmd.append(BaseTest.getProjectRoot())
+        .append("/integration-tests/src/test/resources/statedump.sh");
+    logger.info("Running " + cmd);
+
+    ExecResult result = ExecCommand.exec(cmd.toString());
+    if (result.exitValue() == 0) {
+      logger.info("Executed statedump.sh " + result.stdout());
+    } else {
+      logger.info("Execution of statedump.sh failed, " + result.stderr() + "\n" + result.stdout());
+    }
+
+    // if (JENKINS) {
+    result = cleanup();
```

Comment:
Always calling cleanup on test completing is a relatively big change in some ways.  If nothing else, it adds more time to the end of the run, plus makes it harder to diagnose a failed run.    Is this needed on Wercker and locally?

Reply:
On Wercker, currently only ITOperator class is executed so I don't think its needed. Locally also, I doubt if its needed, I can try by skipping cleanup locally and see.

Reply:
Done